toolkit/authentication/src/main/java/com/arcgismaps/toolkit/authentication/Extensions.kt

```diff
@@ -97,14 +97,14 @@ public fun Activity.launchCustomTabs(pendingBrowserAuthenticationChallenge: Brow
  *
  * @param authorizeUrl the authorize URL used by the custom tabs browser to prompt for OAuth
  * user credentials.
- * @param useIncognito whether the [CustomTabsIntent] should use Incognito mode, if available.
+ * @param preferPrivateWebBrowserSession whether the [CustomTabsIntent] should use a private web browser session, if available.
  *
  * @since 200.2.0
```

Comment:
Should these `since` tags now be updated as these would be cherry picked?
```suggestion
 * @since 200.8.1
```